### ISSUE TITLE
[KEYBOARD]: Skip to Content Link doesn't work correctly in IE11

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2246]
## Description

Our Skip to Content link is designed to let users skip past the initial global navigation and headlines, directly to the main content area. IE11 does not focus the content container as expected, and our skip link should be considered broken for IE11 users who prefer to navigate by keyboard. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As an IE11 keyboard user, I want to press `ENTER` on our Skip to content link, and be able to press `TAB` again and have focus settle on the correct content.
* As a non IE user, I do not want my visual or aural experience to be broken or changed.

## Environment

* Windows 10
* IE11
* staging.va.gov static pages (we are often managing focus on applications)

## Steps to Recreate

1. Enter `https://staging.va.gov/` in IE11
2. Press `TAB` once to highlight the Skip to content link
3. Press `ENTER` on the skip link
4. Press `TAB` again. Verify the yellow focus halo is **not** on the Service member benefits heading, which it should be.

## Possible Fixes (optional)

Adding a `tabindex="-1"` to the target container seems to fix the issue. I am not concerned if we use a script or hard code this value as long as it does not interfere with user experience in non-IE11 browsers.

```html
<section id="content" tabindex="-1">
  ...
</section>
```

**Prior Art**
* https://github.com/uswds/uswds-site/pull/230
* https://github.com/Automattic/_s/blob/master/js/skip-link-focus-fix.js

## WCAG or Vendor Guidance (optional)

* [Understanding Success Criterion 2.4.1 (W3C)
](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-03 at 5 00 02 PM](https://user-images.githubusercontent.com/934879/66167761-b609e100-e600-11e9-979b-c8f4bbf06c78.png)

**Comment by cvalarida** [comment]
If it's as simple as adding a `tabindex=-1`, I think any FE could make this change, but given that it'll be in the content template, maybe Global UX makes the most sense for this one.

**Comment by alexpappasoddball** [comment]
@Karak888 Sending this one your way as we think it would be in your teams wheelhouse.  Let me know if you have any questions.

**Comment by Karak888** [comment]
@alexpappasoddball we are unclear as to what this issue is, specifically in the "content template"

**Comment by 1Copenut** [comment]
@Karak888 content template refers to the Markdown file(s) we're using to generate static HTML pages. I *think* there's just one that is being used as a blueprint file for all of the variations. If that's the case, this should be a one-file fix, but it will need QA testing across lots of template types with IE11 to ensure proper functionality.

**Comment by 1Copenut** [comment]
@brianalloyd This is both duplicated, and older (wrong) guidance where to set focus. I'm going to close this one and comment on the new issue Josh wrote. Thank you for the heads up.
